versions.props

```diff
@@ -30,5 +30,5 @@ org.slf4j:* = 1.7.32
 
 # dependency-upgrader:OFF
 # Explicitly maintaining back-compat with dropwizard metrics
-io.dropwizard.metrics:* = 3.2.6
+io.dropwizard.metrics:* = 4.1.1
```

Comment:
Can we recommend latest 4.1.x? Can we update the comment if we have to pin minimum version to 4.1.1 due to Spark (seems like https://github.com/palantir/spark/blob/6e3520d7625749ecc20a4892282ac75950fb1cfa/pom.xml#L157 )? Are we assuming non-Spark down streams will pull latest 4.1.x ([currently 4.1.26](https://github.com/dropwizard/metrics/releases)) since we're not taking auto-bumps in tritium?


```suggestion
io.dropwizard.metrics:* = 4.1.26
```

Reply:
On the Spark side, we may want to consider backporting a few commits (and possibly bump to `4.2.4`):

- https://issues.apache.org/jira/browse/SPARK-35487 : https://github.com/apache/spark/commit/6bd6e46aeccf2f0706461dd5a66ecb53b5ec38ef
- https://issues.apache.org/jira/browse/SPARK-36013 : https://github.com/apache/spark/commit/7f703509297fb5b6984a99e6f3bb96b77ca74b10